tests/copilot_helper/test_command_application.py

```diff
@@ -0,0 +1,593 @@
+from unittest.mock import patch
+
+import boto3
+from click.testing import CliRunner
+from moto import mock_ecs
+from moto import mock_logs
+from moto import mock_sts
+
+from dbt_copilot_helper.commands.application import get_query_results
+from dbt_copilot_helper.commands.application import stats
+from dbt_copilot_helper.commands.application import stats_long
+
+
+@mock_logs
+@mock_ecs
+@mock_sts
+def test_get_query_results(alias_session):
+    session = boto3.Session(region_name="eu-west-2")
+    session.client("ecs").create_cluster(clusterName="testapp-dev-Cluster-blah")
+    session.client("ecs").create_cluster(clusterName="testapp-test-Cluster-blah")
+
+    # Currently get_query_results is not returning anything for our query, as per doc
+    # not all query results are implemented.   http://docs.getmoto.org/en/latest/docs/services/logs.html
+    # So for now we will only check for 200 success code.
+    #
+    session.client("logs").create_log_group(
+        logGroupName="/aws/ecs/containerinsights/testapp-test-Cluster-blah/performance"
+    )
+    session.client("logs").create_log_stream(
+        logGroupName="/aws/ecs/containerinsights/testapp-test-Cluster-blah/performance",
+        logStreamName="FargateTelemetry-7624",
+    )
+    message = '{"Version":"0","Type":"Task","TaskId":"3f6db7f58bd145c7a668f562ff896fb7","TaskDefinitionFamily":"testapp-test-web","TaskDefinitionRevision":"2","ServiceName":"testapp-test-web-Service-blah","ClusterName":"testapp-test-Cluster-blah","KnownStatus":"RUNNING","CreatedAt":1707384290000,"StartedAt":1707384290000,"Timestamp":1707384300000,"CpuUtilized":23.619295457204185,"CpuReserved":256,"MemoryUtilized":277,"MemoryReserved":1024,"StorageReadBytes":138981376,"StorageWriteBytes":1806336,"NetworkRxBytes":2198,"NetworkRxPackets":3125725,"NetworkTxBytes":1005,"NetworkTxPackets":3039639}'
+    session.client("logs").put_log_events(
+        logGroupName="/aws/ecs/containerinsights/testapp-test-Cluster-blah/performance",
+        logStreamName="FargateTelemetry-7624",
+        logEvents=[
+            {"timestamp": 1707469200000, "message": message},
+        ],
+    )
+    result = get_query_results("test", "testapp", "foo", "query_string")
+
+    assert result["ResponseMetadata"]["HTTPStatusCode"] == 200
+
+
+@mock_logs
+@mock_ecs
+@mock_sts
+@patch(
+    "dbt_copilot_helper.commands.application.get_query_results",
+    return_value={
```

Comment:
At a glance it looks like this `return_value` is repeated several times in this file. Can it be extracted to a variable and reused?

Reply:
correct, moved out to VAR.